content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/new_delivery_method.rst

```diff
@@ -14,34 +24,35 @@ Methods`.
       by checking the corresponding checkbox.
 
    .. image:: new_delivery_method/enable-delivery.png
-      :alt: Enable the *Delivery Methods* feature by checking the box in Configuration > Settings.
+      :alt: Enable the Delivery Methods feature by checking the box in Configuration > Settings.
```

Comment:
Avoid repeating the previous instruction in the alt tag– rather describe the image :)
```suggestion
      :alt: The Delivery Methods feature enabled in the settings.
```